packages/klevu-ui/src/components/klevu-search-landing-page/klevu-search-landing-page.tsx

```diff
@@ -217,7 +217,7 @@ export class KlevuSearchLandingPage {
       if (this.showPriceAsSlider === undefined) {
         this.showPriceAsSlider = settings.klevu_showPriceSlider
       }
-      if (this.priceInterval === undefined) {
+      if (this.priceInterval === undefined && typeof settings.klevu_uc_userOptions.priceInterval !== "undefined") {
```

Comment:
why "typeof" is needed here, can optional chaining used?

Reply:
added undefined check since its an integer and to make sure parseInt works.